cmd/kpod/spec.go

```diff
@@ -372,24 +382,25 @@ func (c *createConfig) GetVolumeMounts() ([]spec.Mount, error) {
 			options = strings.Split(spliti[2], ",")
 		}
 		options = append(options, "rbind")
-		// var foundrw, foundro,
-		var foundz, foundZ bool
+		var foundrw, foundro, foundz, foundZ bool
+		var rootProp string
 		for _, opt := range options {
 			switch opt {
-			// case "rw":
-			// 	foundrw = true
-			// case "ro":
-			// 	foundro = true
+			case "rw":
+				foundrw = true
```

Comment:
I am adding this back because I noticed that even though it is mounted as rw by default, "rw" does not show up in the options of the mount if not added here.
```
            {
                "destination": "/run/test",
                "type": "bind",
                "source": "/home/umohnani/go",
                "options": [
                    "Z",
                    "rbind",
                    "rprivate"
                ]
            }
```
vs
```
{
                "destination": "/run/test",
                "type": "bind",
                "source": "/home/umohnani/go",
                "options": [
                    "Z",
                    "rbind",
                    "rw",
                    "rprivate"
                ]
            }
```